lib/util.js

```diff
@@ -127,40 +128,28 @@ function styleText(format, text, { validateStream = true, stream = process.stdou
     ) {
       throw new ERR_INVALID_ARG_TYPE('stream', ['ReadableStream', 'WritableStream', 'Stream'], stream);
     }
-  }
-
-  if (ArrayIsArray(format)) {
-    let left = '';
-    let right = '';
-    for (const key of format) {
-      const formatCodes = inspect.colors[key];
-      if (formatCodes == null) {
-        validateOneOf(key, 'format', ObjectKeys(inspect.colors));
-      }
-      left += escapeStyleCode(formatCodes[0]);
-      right = `${escapeStyleCode(formatCodes[1])}${right}`;
-    }
 
-    return `${left}${text}${right}`;
+    // If the stream is falsy or should not be colorized, set skipColorize to true
+    skipColorize = !lazyUtilColors().shouldColorize(stream);
```

Comment:
How about returning `text` here if `lazyUtilColors().shouldColorize(stream)` is false?
(If this change is applied, we also need to modify line 122, 147, and 152)
```suggestion
    if (!lazyUtilColors().shouldColorize(stream)) {
      return text;
    }
```

Reply:
This would not validate the styles which is what I wanted to avoid